cves/CVE-2017-11665.yml

```diff
@@ -90,19 +92,12 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: '08c073434e25cba8c43aae5ed9554fdd594adfb0'
-  note: ''
+  note: 'Converted ff_amf_tag_size() to bytestream2'
```

Comment:
What does this mean? If it's just copying from the commit message, we have that already. Same with others.